ui/app/components/tx-list.js

```diff
@@ -63,71 +70,24 @@ TxList.prototype.renderTransactionListItem = function (transaction) {
     address: transaction.txParams.to,
     transactionStatus: transaction.status,
     transactionAmount: formatBalance(transaction.txParams.value, 6),
+    transActionId: transaction.id,
+  }
+  const { address, transactionStatus, transactionAmount, dateString, transActionId } = props
+  const { showConfTxPage } = this.props
+
+  const opts = {
+    transactionStatus,
+    transActionId,
+    dateString,
+    address,
+    transactionAmount,
   }
-  const { address, transactionStatus, transactionAmount, dateString } = props
-
-  return h('div', {}, [
-    h('div.flex-column.tx-list-item-wrapper', {
-      style: {}
-    }, [
-
-      h('div.tx-list-date-wrapper', {
-        style: {}
-      }, [
-        h('span.tx-list-date', {}, [
-          dateString,
-        ])
-      ]),
-
-      h('div.flex-row.tx-list-content-wrapper', {
-        style: {}
-      }, [
-
-        h('div.tx-list-identicon-wrapper', {
-          style: {}
-        }, [
-          h(Identicon, {
-            address,
-            diameter: 28,
-          })
-        ]),
-
-        h('div.tx-list-account-and-status-wrapper', {}, [
-          h('div.tx-list-account-wrapper', {
-            style: {}
-          }, [
-            h('span.tx-list-account', {}, [
-              `${address.slice(0, 10)}...${address.slice(-4)}`
-            ]),
-          ]),
-
-          h('div.tx-list-status-wrapper', {
-            style: {}
-          }, [
-            h('span.tx-list-status', {}, [
-              transactionStatus,
-            ]),
-          ]),
-        ]),
-
-        h('div.flex-column.tx-list-details-wrapper', {
-          style: {}
-        }, [
-
-          h('span.tx-list-value', {}, [
-            transactionAmount,
-          ]),
-
-          h('span.tx-list-fiat-value', {}, [
-            '+ $300 USD',
-          ]),
-
-        ]),
 
-      ]),
-    ]),
-    contentDivider
+  if (transactionStatus === 'unapproved') {
+    opts.onClick = () => showConfTxPage({id: transActionId})
```

Comment:
why do we show confirmation page only when the transaction is unapproved? Shouldn't we only do so if the tx is approved?

Reply:
I thought the confirm page was where the user is able to review their transaction and change its status from `'unapproved'` to `'confirmed'` by clicking the "Confirm" button?